Model/LdapUsers.php

```diff
@@ -383,16 +383,17 @@ public function getUserEntryQuery($username = null)
     {
         $bind = array();
         $conditions = array();
-
+        $memberOfField = Config::getRequiredMemberOfField();
```

Comment:
We shouldn't access the new config option directly here. Like the other options there should be a private member, and public getter & setter. The field should be initialized in the LdapUsers::makeConfigured function.